Vagrantfile

```diff
@@ -62,7 +58,7 @@ Vagrant.configure("2") do |config|
   config.vm.box = "archlinux64"
   config.vm.box_check_update = true
 
-  config.vm.network "private_network", ip: "192.168.33.5", auto_config: false
```

Comment:
There are other places in the code, for example the installer, where the value 33 is used as the default. If this is not necessary, it's probably easier to just leave this. Otherwise, I think it should be changed everywhere.